service/prometheus/prometheus.go

```diff
@@ -9,58 +10,107 @@ const (
 	// ClusterAnnotation is the Kubernetes annotation that identifies Services
 	// that the prometheus-config-controller should scrape.
 	ClusterAnnotation = "giantswarm.io/prometheus-cluster"
+)
+
+// Prometheus Kubernetes service discovery labels.
+var (
```

Comment:
I cleaned up all the constants, as they pretty much all moved around. Hopefully it's clearer now.